diff -Naur -x '*.orig' Params-Validate-0.98/lib/Params/Validate.xs Params-Validate-0.98-fix-format-errors//lib/Params/Validate.xs
--- Params-Validate-0.98/lib/Params/Validate.xs	2011-04-23 19:06:15.000000000 +0200
+++ Params-Validate-0.98-fix-format-errors//lib/Params/Validate.xs	2011-05-02 19:42:18.000000000 +0200
@@ -1007,7 +1007,7 @@
                             sv_catpv(buffer, "' does not exist in spec: ");
                             sv_catsv(buffer, depend_name);
 
-                            croak(SvPV_nolen(buffer));
+                            croak("%s", SvPV_nolen(buffer));
                         }
                         /* if we got here, the spec was correct. we just
                          * need to issue a regular validation failure
